### PR TITLE
Add backup_restore vm not enabled troubleshooting

### PR DESCRIPTION
**Pull request by neil-hickey**

### CODE REVIEW

File: install/backup-restore/troubleshooting-bbr.html.md.erb [hunk 1]
```diff
@@ -157,3 +157,18 @@ There is a problem with your deployment install.
 #### Solution
 
 Contact [Support](https://support.pivotal.io/).
+
+### <a id="metadata-error"></a> Backup Restore VM not enabled
+
+#### Symptom
+
+1. BBR backup is small in size.
+1. BBR backup is missing artifacts for deployed applications on TAS.
```
Comment (tinygrasshopper) [inline install/backup-restore/troubleshooting-bbr.html.md.erb:166]: Another symptom is., if you `tar tvf` on the backup file, you cant see the `backup-restore` folder

File: install/backup-restore/troubleshooting-bbr.html.md.erb [hunk 2]
```diff
@@ -157,3 +157,18 @@ There is a problem with your deployment install.
 #### Solution
 
 Contact [Support](https://support.pivotal.io/).
+
+### <a id="metadata-error"></a> Backup Restore VM not enabled
+
+#### Symptom
+
+1. BBR backup is small in size.
```
Comment (tinygrasshopper) [inline install/backup-restore/troubleshooting-bbr.html.md.erb:165]: > backup is small in size
That is actually not true. The backup contains the copy of the blobstore, so the backup will be the similar size if the vm was enabled.

Comment (neil-hickey) [reply install/backup-restore/troubleshooting-bbr.html.md.erb:165]: gotcha, makes sense. I think I wanted to say something nuanced like, it's smaller than it should be.

But I can't know what the "should" is. Maybe I will ignore this, outlining the symptoms is hard when they aren't really obvious until you do a restore.

**Comment by cshollingsworth** [comment]
@neil-hickey This pull request draft was last touched over a year ago. Do you and your team still intend to work on this, or should this PR be closed?

**Pull request closed without merging**